vagrant/provisioning/roles/nmos-joint-ri-setup/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+# module: nmos-joint-ri-setup/tasks
+# description: Provision NMOS Joint RI VM
+
+- name: Install all required packages for nmos-joint-ri-setup
+  apt: pkg={{ packages }} state=present
+
+- name: Install Apache2 modules
+  apache2_module:
+    state: present
+    name: "{{ item }}"
+  with_items: 
```

Comment:
Perhaps the Apache modules could be added to the defaults?

Reply:
Done